app/build.gradle

```diff
@@ -22,14 +28,100 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+
+    flavorDimensions "app"
+    productFlavors {
+        develop {
+            dimension "app"
+            versionNameSuffix "-dev"
+            buildConfigField 'String', 'API_URL', '"https://api.exchangerate.host/"'
+            buildConfigField 'String', 'BALANCE_NOTIFICATION_URL', '"http://10.0.2.2:3000/"'
+        }
+        mock {
+            dimension "app"
+            versionNameSuffix "-mock"
+            buildConfigField 'String', 'API_URL', '"https://my_api_url.com/"'
+        }
+        prod {
+            dimension "app"
+            buildConfigField 'String', 'API_URL', '"https://my_api_url.com/"'
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+
+repositories {
+    maven { url 'https://jitpack.io' }
 }
 
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+
+    //UI
```

Comment:
Remove dependencies you're not using right now, keep the project as clean as posible